### DISCUSSION TITLE
Sidebar autohide when menu item clicked in mobile

### DISCUSSION THREAD

**Discussion by icm-app** [vikramlearning/blazorbootstrap#898] [Q&A]
Hi Vikram, 
Is there any setup in sidebar2, so when in mobile browser, the sidebar auto hide when we clicked the item? 

Thanks,
Dodo

**Comment by ecospecifier** [comment]
![image](https://github.com/user-attachments/assets/19d9c7d4-3bd7-4202-96b2-575d7dd97424)

I Did it by doing the sidebar in an off canvas setup and adding data-bs-dismiss="offcanvas" to the Sidebar component
